plugins/org.eclipse.n4js.ui/src/org/eclipse/n4js/ui/MyReferenceSearchResultContentProviderCustomModule.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2018 NumberFour AG.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   NumberFour AG - Initial API and implementation
+ */
+package org.eclipse.n4js.ui;
+
+import org.eclipse.n4js.ui.search.MyReferenceSearchResultContentProvider;
+import org.eclipse.xtext.ui.editor.findrefs.ReferenceSearchResultContentProvider;
+import org.eclipse.xtext.ui.editor.findrefs.ReferenceSearchResultLabelProvider;
+import org.eclipse.xtext.ui.editor.findrefs.ReferenceSearchViewPage;
+import org.eclipse.xtext.ui.editor.findrefs.ReferenceSearchViewSorter;
+
+import com.google.inject.AbstractModule;
+import com.google.inject.PrivateModule;
+
+/**
+ * This overriding module binds ReferenceSearchResultContentProvider to a custom MyReferenceSearchResultContentProvider
+ * that aims to fix GH-724.
+ */
+@SuppressWarnings("restriction")
+public class MyReferenceSearchResultContentProviderCustomModule extends AbstractModule {
+
+	@Override
+	protected void configure() {
+		binder().install(new PrivateModule() {
+			@Override
+			protected void configure() {
```

Comment:
Is it possible to somehow define the binding to `MyReferenceSearchResultContentProvider` without copying the other bindings (e.g. `ReferenceSearchViewPage`).

If not (which I guess is the case), then at least add a comment that you here copy the defaults defined in `org.eclipse.xtext.ui.shared.internal.SharedModule` (at the bottom) and that the following has to be kept in line with those defaults.

Reply:
Done. Comments added.